clustering/spi/src/main/java/org/wildfly/clustering/spi/GroupServiceName.java

```diff
@@ -40,4 +40,15 @@ public String toString() {
             return "group";
         }
     },
+    GROUP_CACHE() {
```

Comment:
What is this for?  This change shouldn't have anything to do with the clustering api.